streaming/README.md

```diff
@@ -511,4 +511,99 @@ metrics, following the same pattern of the receiving side.
+Y unsent - we have sent Z bytes in total, W on this connection. Closing connection to flush the data.
+```
+
+where the value X, Y, Z and W are integer numbers.
+
+On the other hand, in the master side we can have different reports where the most common message that you can see is
+
+```
+netdata ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : read failed: end of file
+```
+
+Another common problem that can happen is the slave sends only part of the message for the master, in this case the master
+will not have condition to understand the message received and it will write inside error.log
+
+```
+ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : sent command 'B' which is not known by netdata, for host 'HOSTNAME'. Disabling it.
+```
+
+in this example, 'B' was part of a 'BEGIN' message that was cut due connection problems.
+
+The last problem related to slow connections happens when some message was missed and the following commands received had a
+direct connection with it, for example, when the slave could not send the list of charts available on its host and it
+sends in the sequence a 'SET' message on a chart that was missed, the master will show a messages like this
+
+```
+ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : requested a SET on chart 'CHART NAME' of host 'HOSTNAME', without a dimension. Disabling it.
+```
+
+### Slave cannot connect to master
+
+Sometimes we are type writing the address and port of the master inside stream.conf and we write a wrong information, Netdata will write a message like this in the slave log
+
+```
+ERROR : STREAM_SENDER[HOSTNAME] : Failed to connect to 'MASTER IP', port 'MASTER PORT' (errno 113, No route to host)
+```
+
+this error can also happen case the firewall on master is not allowing connections to the master port.
+
+### Is this a Netdata?
+
+The question that names this section can appear when Netdata is starting the stream, this happens when the slave does not receive the
+expected return for the beginning of the communication. There are different motives for this happen and the main motive among them is the fact that master was
+configured with SSL activated while the slave tried to send a plain text information. The complete error message will have the format
+
+```
+ERROR : STREAM_SENDER[SLAVE HOSTNAME] : STREAM SLAVE HOSTNAME [send to MASTER HOSTNAME:MASTER PORT]: server is not replying properly (is it a netdata?).
+```
+
+### Stream charts wrong
+
+When the slave is sending data to master it cannot have a different chart that it has locally, case there are differences between the charts, this can mean that your master
+memory mode is not configured correctly.
+
+### Forbidding access
+
+These are error reported in the master side, they can also happen because there is a slow connection between master and slave, but this
```

Comment:
> If the master reports errors about forbidding access, it could be because of a slow connection between the master and slave nodes. But, there are other reasons for this error. The complete message you would see in your log file would look something like this:

Reply:
I overwrite the whole text with your suggestion.